### ISSUE TITLE
GPC signal is sent despite being disabled on Firefox

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/gpc-optmeowt#203]
It looks like if I disable the GPC signal sending for an individual domain, it is still being sent on Firefox. I saw this behavior on citi.com on the most recent version of Firefox. Do you have the same issue, @kalicki1? If so, could you look into it? 

<img width="897" alt="Screen Shot 2021-08-02 at 10 42 35 AM" src="https://user-images.githubusercontent.com/11034003/127902797-74ffe0f0-9409-4a14-90c7-1db7d53c8f80.png">

**Comment by kalicki1** [comment]
@SebastianZimmeck, I took a cursory look at this and did not see an issue with how the signals are sent. I'm using the most recent version of Firefox and the most recent main branch commit. I ran tests on our GPC test server and citi.com. Whenever I disabled the extension or the individual domain, the signal would be turned off. Let me know if you are still having this issue. 

I'm not sure if this is what you did when you tested, but in this version the logic is setup such that if you disable a signal, the `domainlist` mode is turned on. However, if you then `disable` the extension and then re-`enable` it, the mode switches to `enable` instead of back to `domainlist`. This then overrides the individual domain off switch. Let me know if this is what you did. I think I mentioned briefly in some old issue we should include a 3-way switch or an indicator denoting whether individual domains are being selected at some point in the future.
